src/main/python/hydra/lib/mmapi.py

```diff
@@ -105,6 +107,30 @@ def update_slaves(self):
             self.slavesHN[itm['hostname']] = itm
             self.slavesID[itm['id']] = itm
 
+    def get_health(self):
```

Comment:
I think all lib functions should have proper documentation, this will help in building docs for api later.